salesagility/SuiteCRM
# Make sure deleted users are not used when validating users

Closes #3440 

## Description
When the user validates its login now it discards deleted users thanks to adding deleted=0 to the mysql query.

## Motivation and Context
After deleting and recreating a user then, depending on how mysql tables is handled then users cannot longer login. You can see similar problems when you try to reset the password and it does not work.

## How To Test This
Try to add duplicated users entries in the mysql users table where you change deleted=0 into deleted=1 and see if anyone of them is taken into account when logging it. If it is not then the bug has been succesfully fixed.

## Types of changes


### Final checklist
